src/components/task-plan/homework/exercises.cjsx

```diff
@@ -184,8 +185,10 @@ ExerciseTable = React.createClass
 
   renderExerciseRow: (exerciseId, index, hasTeks) ->
     {section, lo, tagString} = ExerciseStore.getTagStrings(exerciseId)
-    content = ExerciseStore.getContent(exerciseId)
-
+    content = $("<span>#{ExerciseStore.getContent(exerciseId)}</span>")
```

Comment:
It was showing up as a string, not an image when added to the table, so the
user was seeing html: <img src....etc.

On Wed, Jun 3, 2015 at 6:29 AM, Philip Schatz <notifications@github.com>
wrote:

> In src/components/task-plan/homework/exercises.cjsx
> <https://github.com/openstax/tutor-js/pull/361#discussion_r31578121>:
>
> > @@ -184,8 +185,10 @@ ExerciseTable = React.createClass
> >
> >    renderExerciseRow: (exerciseId, index, hasTeks) ->
> >      {section, lo, tagString} = ExerciseStore.getTagStrings(exerciseId)
> > -    content = ExerciseStore.getContent(exerciseId)
> > -
> > +    content = $("<span>#{ExerciseStore.getContent(exerciseId)}</span>")
>
> Why not just hide it via CSS?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openstax/tutor-js/pull/361/files#r31578121>.
>